routes/api.php

```diff
@@ -240,6 +240,18 @@
     'uses' => 'Api\Chat\ChatController@getChatMessages',
 ]);
 
+Route::delete('v1/users/message/{message}', [
+    'middleware' => ['jwt.auth'],
+    'as' => 'delete.message',
+    'uses' => 'Api\Chat\ChatController@destroy',
+]);
+
+Route::put('v1/users/message/{message}/mark', [
```

Comment:
change endpoint. Use for example `'v1/users/message/{message}/is_read'`

Reply:
@sergey-ben fixed it.